lib/core/presentation/pages/edit_profile/sub_pages/edit_profile_personal_page.dart

```diff
@@ -103,86 +111,43 @@ class EditProfilePersonalDialog extends StatelessWidget with LemonBottomSheet {
                                   .map((e) => e.industry)
                                   .toList(),
                               onValueChange: bloc.onIndustrySelect,
-                              selectedValue:
-                                  bloc.state.industry ?? userProfile.industry,
+                              selectedValue: bloc.state.industry ??
+                                  widget.userProfile.industry,
                             ),
                             SizedBox(height: Spacing.smMedium),
-                            LemonTextField(
-                              label: t.profile.educationQualification,
-                              onChange: bloc.onEducationChange,
-                              initialText: userProfile.education,
+                            EditProfileFieldItem(
+                              profileFieldKey: ProfileFieldKey.educationTitle,
+                              userProfile: widget.userProfile,
+                              onChange: bloc.onOrganizationChange,
                             ),
                             SizedBox(height: Spacing.smMedium),
-                            FrostedGlassDropDownV2(
-                              label: t.profile.gender,
-                              hintText: t.profile.hint.gender,
-                              listItem: LemonGender.values
-                                  .map((e) => e.newGender)
-                                  .toList(),
-                              onValueChange: bloc.onGenderSelect,
-                              selectedValue:
-                                  bloc.state.gender ?? userProfile.gender,
+                            EditProfileFieldItem(
+                              profileFieldKey: ProfileFieldKey.newGender,
+                              userProfile: widget.userProfile,
+                              onChange: bloc.onGenderSelect,
+                              selectedValue: bloc.state.gender ??
```

Comment:
why use both data from bloc.state and widget.userProfile ? If you wanna add default value, should populate userProfile data to profile bloc and use only 1 data source in bloc